lib/sundials-config/prepare.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SUNDIALS_VERSION="4.0.1"
+
+SUNDIALS_TAR=sundials-${SUNDIALS_VERSION}.tar.gz
+
```

Comment:
Here, you could do something like `git rm lib/sundials-*` to mark all of the files that we need to delete.